linux/coalescer.go

```diff
@@ -58,6 +58,13 @@ func (c *Coalescer) Coalesce(ctx context.Context, layerArtifacts []*indexer.Laye
 		for db, pkgs := range tmp {
 			dbs[db] = pkgs
 		}
+
+		if len(artifacts.Files) > 0 {
+			c.ir.Files = make(map[string][]claircore.File)
+		}
```

Comment:
I just realized, what happens if two layers have `len(artifacts.Files) > 0`? Seems like we may overwrite `c.ir.Files`. I lead you astray my b

Reply:
Also, RHEL has its own coalescer. Did you mean to update that one?

Reply:
It needed to be updated as the `IndexReport` was updated.

Reply:
Don't we still need to update the RHEL coalescer since this'll affect RHEL images?

Reply:
The RHEL coalescer doesn't come into contact with any RPM files, the linux coalescer is responsible for that as it is run as part of the RPM ecosystem (that includes the rpm.FileScanner). i.e. There files we're finding aren't RHEL specific but RPM specific.